### PR TITLE
Yarn Package Manager

### PR DESCRIPTION
**Pull request by protonate**
## Type of change
- [x] Build related changes

## Description of change
Adds the Yarn package manager. NOTE: Yarn must be installed independently of this change and as fits the developer's environment. See [install guide](https://yarnpkg.com/en/docs/install).

[Usage](https://yarnpkg.com/en/docs/usage) is similar to NPM.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,7825 @@
...
+ansi-escape-sequences@^2.2.1, ansi-escape-sequences@^2.2.2:
+  version "2.2.2"
+  resolved "https://npm.corp.appnexus.com/ansi-escape-sequences/-/ansi-escape-sequences-2.2.2.tgz#174c78d6f8b7de75f8957ae81c7f72210c701635"
+  dependencies:
+    array-back "^1.0.2"
+    collect-all "~0.2.1"
+
+ansi-escape-sequences@^3.0.0:
+  version "3.0.0"
+  resolved "https://npm.corp.appnexus.com/ansi-escape-sequences/-/ansi-escape-sequences-3.0.0.tgz#1c18394b6af9b76ff9a63509fa497669fd2ce53e"
```
Comment (mkendall07) [inline yarn.lock]: Looks to be a lot of internal npm references here. Need to fix.

Comment (protonate) [inline yarn.lock]: Do you mean npm registry references? That is expected, Yarn is an npm registry client, not a replacement. `registry.yarnpkg.com` is a proxy for `registry.npmjs.org`.

**Comment by protonate** [comment]
@mkendall07 better now

**Comment by mkendall07** [comment]
LGTM.

**Comment by mkendall07** [comment]
@snapwich 
please review

File: README.md [hunk 2]
```diff
@@ -65,18 +61,18 @@ Building with just these adapters will result in a smaller bundle which should a
 
 **Build standalone prebid.js**
 
-- Clone the repo, run `npm install`
+- Clone the repo, run `yarn install`
```
Comment (snapwich) [inline README.md]: Might want to add a link to the yarn project for those that are unfamiliar with it.

**Comment by protonate** [comment]
Note addressed, thanks for reviews.

**Pull request merged**